cmd/ddev/cmd/sequelpro.go

```diff
@@ -26,8 +26,8 @@ var SequelproLoc = "/Applications/sequel pro.app"
 // DdevSequelproCmd represents the sequelpro command
 var DdevSequelproCmd = &cobra.Command{
 	Use:   "sequelpro",
-	Short: "Easily connect a dev site to sequelpro",
-	Long:  `A helper command for easily using sequelpro (OSX database browser) with a running ddev app.`,
+	Short: "Connect a project database to sequelpro",
```

Comment:
I don't think this should hold up the PR, but I think something to the effect of "Connect to a project database with Sequel Pro" would be more accurate.